Makefile

```diff
@@ -123,6 +127,8 @@ define do-docker-retag-and-push-public
 endef
 
 .PHONY: docker-retag-and-push-public
+docker-retag-and-push-public: export DOCKER_HOSTNAME?=kind-registry:5000
```

Comment:
Why is this changed? Shouldn't it stay as the public vars?

Reply:
removed it from the .travis env to makefile (thats' all) so that when running the command from the makefile it runs as it should